tests/indexes_/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Article(models.Model):
+    headline = models.CharField(max_length=100)
+    pub_date = models.DateTimeField()
+    published = models.BooleanField(default=False)
+    number = models.IntegerField()
+    text = models.CharField(max_length=100)
+
+    class Meta:
+        indexes = [models.Index(fields=["headline", "pub_date"])]
```

Comment:
probably none of the tests require this index?

Reply:
Yes, they aren't needed, forgot to remove.